src/pages/admin/quizzes/QuizList/components/index.js

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect } from 'react';
```

Comment:
put this component in another folder. not just in the /components folder.

for example /components/QuizModal/index.js